api/Document.json

```
@@ -7651,7 +7651,8 @@
               "version_added": "33"
             },
             "edge": {
-              "version_added": "18"
+              "version_added": "18",
+              "notes": "Before Edge 18, this event handler was not supported, however the event itself was supported since Edge 12. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```

```suggestion
              "notes": "Before Edge 18, this event handler attribute was not supported, however the event itself was supported since Edge 12. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```

I would call it an "event handler attribute" to be as specific as possible. There's also a note for api.Element.wheel_event for a very similar situation &ndash; it would be good to use similar language for both, and for the likely many future cases we'll find.